corehq/apps/hqwebapp/templates/hqwebapp/partials/ko_inline_edit.html

```diff
@@ -12,7 +12,7 @@
       <!-- ko if: lang -->
       <span class="btn btn-xs btn-info btn-langcode-preprocessed" data-bind="text: lang, visible: !value()"></span>
       <!-- /ko -->
-      <span data-bind="text: value, attr: {'class': containerClass + ' ' + readOnlyClass + ' text'}"></span>
+      <span data-bind="text: value, visible: value, attr: {'class': containerClass + ' ' + readOnlyClass + ' text'}"></span>
```

Comment:
The `readOnlyClass` for case property descriptions has a static width, so when there's no value we really don't want to show this span (it pushes the placeholder & pencil icon over to the right).